hack/update-all.sh

```diff
@@ -54,7 +54,7 @@ if ! $ALL ; then
 	echo "Running in short-circuit mode; run with -a to force all scripts to run."
 fi
 
-kube::util::ensure_godep_version v74
+kube::util::ensure_godep_version v79
```

Comment:
Do we really want to set v79 so many places? That is so many places where it could drift, and I'd rather everything use the same version...   Maybe drop all of the v79's?